src/ConformalTracking.cc

```diff
@@ -1620,11 +1728,18 @@ void ConformalTracking::createTracksNew(UniqueCellularTracks& finalcellularTrack
       // If the weight is != 0 and there is more than one path to follow, branch the track (create a new one for each path)
       int nBranches = cell->getFrom().size();
 
+      streamlog_out(DEBUG8) << "- Cellular track " << itTrack << " has more than one extension" << std::endl;
+
       //     streamlog_out(DEBUG7)<<"- making "<<nBranches<<" branches"<<std::endl;
 
       // For each additional branch make a new track
       for (int itBranch = 1; itBranch < nBranches; itBranch++) {
-        auto branchedTrack = std::unique_ptr<cellularTrack>(new cellularTrack(*(cellularTracks[itTrack].get())));
+        streamlog_out(DEBUG8) << "-- Cellular track " << itTrack << ", extension " << itBranch << std::endl;
+        auto branchedTrack      = std::unique_ptr<cellularTrack>(new cellularTrack(*(cellularTracks[itTrack].get())));
+        auto branchedParentCell = SCell(cell->getFrom()[itBranch]);
+        streamlog_out(DEBUG8) << "-- Added branched parent cell A ([x,y] = [" << branchedParentCell->getStart()->getX()
+                              << ", " << branchedParentCell->getStart()->getY() << "]) - B ([x,y] = ["
+                              << cell->getEnd()->getX() << ", " << cell->getEnd()->getY() << "])" << std::endl;
         branchedTrack->push_back(SCell(cell->getFrom()[itBranch]));
```

Comment:
`branchedTrack->push_back(std::move(branchedParentCell))` (I think that should work, avoids the very temporary shared_ptr, which is the same.)